data/categories.txt

```diff
@@ -12745,6 +13093,7 @@ id:Titik penjemputan
 it:Punto di raccolta
 ja:集荷場
 ko:픽업 장소
+lv:Saņemšanas punkts
```

Comment:
Should be "Preču saņemšanas punkts"

Reply:
Does someone type it like this in the search?